src/Advection/weno_fifth_order.jl

```diff
@@ -46,32 +46,34 @@ struct WENO5 <: AbstractUpwindBiasedAdvectionScheme end
 ##### Jiang & Shu (1996) WENO smoothness indicators. See also Equation 2.63 in Shu (1998).
 #####
 
-@inline left_biased_βx₀(i, j, k, ψ) = @inbounds 13/12 * (ψ[i-1, j, k] - 2ψ[i,   j, k] + ψ[i+1, j, k])^2 + 1/4 * (3ψ[i-1, j, k] - 4ψ[i,   j, k] +  ψ[i+1, j, k])^2
-@inline left_biased_βx₁(i, j, k, ψ) = @inbounds 13/12 * (ψ[i-2, j, k] - 2ψ[i-1, j, k] + ψ[i,   j, k])^2 + 1/4 * ( ψ[i-2, j, k]                 -  ψ[i,   j, k])^2
-@inline left_biased_βx₂(i, j, k, ψ) = @inbounds 13/12 * (ψ[i-3, j, k] - 2ψ[i-2, j, k] + ψ[i-1, j, k])^2 + 1/4 * ( ψ[i-3, j, k] - 4ψ[i-2, j, k] + 3ψ[i-1, j, k])^2
+const two_32 = Int32(2) # hut, hut...
```

Comment:
what's `# hut, hut` supposed to mean? I don't even understand the reference now. Imagine in 14 months when somebody is trying to understand why this `Int32(2)` is needed....

Reply:
That's a reference to American (and possibly Canadian) football.  Quarterbacks say such things.

Reply:
Cool, but what do they mean by that? :)
I accept the reference but I suggest we also add a remark that could be informative when we revisit this piece of code few months down the road. Or even at least informative for the archaeologists of the 34th century when they discover this code emerging out from the ice in the palm trees of the North Pole and they are trying to understand what scientists of 2021 were up to.

Reply:
Oh, it's just a joke. I can remove it.